firmware/application/apps/ui_adsb_rx.cpp

```diff
@@ -107,7 +107,7 @@ void ADSBLogger::log(const ADSBLogEntry& log_entry) {
         log_line += " Type:" + to_string_dec_uint(log_entry.vel_type) +
                     " Hdg:" + to_string_dec_uint(log_entry.vel.heading) +
                     " Spd: " + to_string_dec_int(log_entry.vel.speed);
-
+    if (log_entry.sil != 0) log_line += " Sil:" + to_string_dec_uint(log_entry.sil);
```

Comment:
maybe seperate in two lines
but it's already good enough.
it's just a lil hard to read for n00bs like me